spec/support/matchers/accessibility.rb

```diff
@@ -235,3 +235,14 @@ def expect_page_to_have_no_accessibility_violations(page, validate_markup: true)
   expect(page).to be_uniquely_titled
   expect(page).to have_valid_markup if validate_markup
 end
+
+def expect_page_to_have_skip_link(page, &block)
+  page.active_element.send_keys(:tab)
+  expect(page.active_element).to have_content(t('shared.skip_link'), wait: 5)
+
+  if block.present?
+    page.active_element.send_keys(:enter)
+    page.active_element.send_keys(:tab)
+    yield
```

Comment:
It's not clear exactly to me why passing the block is optional? Both uses of this helper call the block, so it doesn't seem to be an option?

What if we wrapped these two `send_keys` in a separate helper call? Like `send_tab_enter_keys`?

Reply:
that works for me

Reply:
Gave it a shot in 1295c09.